### PR TITLE
Adjust content and example error messaging to match actual issue

### PR DESCRIPTION
**Pull request by wcpines**
@fancyremarker I think this is more what you were looking for.

### CODE REVIEW

**Comment by fancyremarker** [comment]
@wcpines: The main issue here is that the issue of "improper formatting," while valid and worth writing a support article for, is not a cause of browser warnings. It's a separate issue that would result in an error message at the time when a user uploads their certificate.

Let's split this into two articles.
1. I'm getting the error "Validation failed: Private key must match certificate," what should I do?
2. I'm seeing an SSL error (`ERR_CERT_AUTHORITY_INVALID`) in my browser.  What should I do?

If you want, you can add a third article, too:
- I'm seeing an SSL error (`ERR_CERT_COMMON_NAME_INVALID`) in my browser.  What should I do?

But if you do write this article, it should be a different article than the `ERR_CERT_AUTHORITY_INVALID` article, since it's a completely different cause.

**Comment by wcpines** [comment]
I think this fixes (2) above.

**Comment by fancyremarker** [comment]
Looks great. Pedantic point: especially once you add a second article for `ERR_CERT_COMMON_NAME_INVALID`, the "e.g." and "like" here are misleading and should probably be removed. The solution you're providing is for a specific error — `ERR_CERT_COMMON_NAME_INVALID` — and not applicable to other browser SSL errors.

**Comment by wcpines** [comment]
I actually thought the same thing right after I pushed up.  Change made.

**Comment by wcpines** [comment]
@fancyremarker New drafts. Will need revision.

File: data/topics.yml [hunk 1]
```diff
@@ -92,8 +92,12 @@
       url: "paas/zero-downtime-deploys"
     - title: "How can I modify the way my app handles SSL?"
       url: "paas/how-to-modify-nginx-config"
-    - title: "I'm getting the error \"Validation failed: Private key must match certificate,\" what should I do?"
+    - title: "My default vhost's cname is showing an SSL error ('ERR_CERT_COMMON_NAME_INVALID'). What should I do?"
```
Comment (fancyremarker) [inline data/topics.yml]: This phrasing is very confusing. I like the pattern of the next title better:

> I'm seeing an SSL error ('ERR_CERT_COMMON_NAME_INVALID') in my browser. What should I do?

File: source/topics/paas/how-to-cname-default-vhosts.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+This error indicates that the CNAME requested does not match the associated host. This often happens when one has associated a new domain (e.g. myapp-staging.com) to a default vhost ($APP-ID.on-aptible.com). Aptible has its own wildcard certificate which covers all default vhosts.  This causes a browser to throw an error when the requested domain does not match.
+
+Instead, provision a separate custom vhost for the app in question, and map this to your desired hostname using your DNS provider of choice.
```
Comment (fancyremarker) [inline source/topics/paas/how-to-cname-default-vhosts.md]: Great. I'd also link to the article explaining how to provision a custom VHOST.

**Comment by fancyremarker** [comment]
Thanks!

**Pull request merged**